instrumentation/apache-dubbo/apache-dubbo-2.7/library/src/main/java/io/opentelemetry/instrumentation/apachedubbo/v2_7/DubboTracer.java

```diff
@@ -46,7 +47,10 @@ public Context startClientSpan(String interfaceName, String methodName) {
 
   public void end(Context context, Result result) {
     Span span = Span.fromContext(context);
-    span.setStatus(DubboHelper.statusFromResult(result));
+    StatusCode statusCode = DubboHelper.statusFromResult(result);
+    if (statusCode != StatusCode.UNSET) {
+      span.setStatus(statusCode);
```

Comment:
can pull `Span span = Span.fromContext(context);` line down from above
```suggestion
      Span.fromContext(context).setStatus(statusCode);
```